airflow/scripts/git_scripts/fetch_repo_data.py

```diff
@@ -0,0 +1,58 @@
+import requests
+
+
+token = 'put_token_here'   # add your token
+
+
+def get_repo_data(owner, repo, token):
```

Comment:
*relevant to all functions* 
Use try and exception to handle response errors or any errors.
Use a logger to log everything (info, warnings, Errors)